dev/com.ibm.ws.jdbc_fat_oracle/fat/src/com/ibm/ws/jdbc/fat/oracle/FATSuite.java

```diff
@@ -28,4 +37,54 @@ public static void beforeSuite() throws Exception {
         //Allows local tests to switch between using a local docker client, to using a remote docker client. 
         ExternalTestServiceDockerClientStrategy.clearTestcontainersConfig();
     }
+    
+    public static void initDatabaseTables(OracleContainer oracle) throws SQLException {
```

Comment:
Minor: mixture of tabs and spaces makes this code for the following method format awkwardly